autonomica-api/app/api/routes/seo.py

```
@@ -0,0 +1,482 @@
+
+This module provides API endpoints for:
+- Keyword research and analysis
+- SERP analysis and competitor research
+- Keyword clustering and opportunity identification
+- SEO scoring and recommendations
+"""
+
+import os
+import logging
+from typing import List, Optional
+from fastapi import APIRouter, HTTPException, Depends, Query
```

Ruff (0.12.2)

`fastapi.Query` imported but unused

Remove unused import: `fastapi.Query`

(F401)